src/Splitio/Domain/SdkInternalEvent.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+namespace Splitio.Domain
+{
+    public enum SdkInternalEvent
+    {
+        FlagsUpdated,
+        FlagKilledNotification,
+        RuleBasedSegmentsUpdated,
+        SegmentsUpdated,
+        LargeSegmentsUpdated
```

Comment:
You may need a TimeoutReached internal event or something later

Reply:
sure, will add